source/installation-guide/wazuh-indexer/step-by-step.rst

```diff
@@ -163,7 +169,7 @@ Starting the service
 Repeat this stage of the installation process for every Wazuh indexer node in your cluster. Then proceed with initializing your single-node or multi-node cluster in the next stage.
 
 
-3. Cluster initialization
+1. Cluster initialization
```

Comment:
Wrong index

```suggestion
3. Cluster initialization
```

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/9b2298127240e77d2df7ffd2b3e3f96e5a990cb1) commit